### PR TITLE
Add User Interface for calculator App

### PR DESCRIPTION
**Pull request by developerwaleed**
## Requirements

Delete all the boilerplate from CRA (text, images, styles). ✅
Create a directory called components.✅
Inside components, create a new Calculator.js file.✅
In Calculator.js, create a React component that matches the given Design.✅
Don't add any extra design.✅
Develop just the UI of the calculator; the math logic to make it work will be added in the next project.✅
Use class based components (All components should be class-based).✅
Once the Calculator is ready, import it and use it in your main component (App.js).✅
Test it in the browser; it should render without problems.✅

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,13 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
+
+// const calculator = new Calculator();
 
 function App() {
   return (
```
Comment (ShadyShawkat) [inline src/App.js]: - [x] Kindly make sure all of the components are implemented as class components, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @caasperr, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
- [ ] Please use [this](https://github.com/microverseinc/readme-template/blob/master/README.md) readme template and edit each section to better suit your project.
- [ ] _Check the comments under the review for more changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (developerwaleed) [reply src/App.js]: hi @ShadyShawkat , Thank you so much for the suggestion I'll definitely apply it I have some questions can we please arrange a little zoom meeting? please Thankyou :)

Comment (ShadyShawkat) [reply src/App.js]: Hey @caasperr, 
Sure, you can reach me on Slack.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @caasperr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**